entrypoint.sh

```diff
@@ -228,6 +232,16 @@ function seiso_lint() {
 			linter["$key"]=$value
 		done < <(echo "$line" | jq -r 'to_entries|map("\(.key)=\(.value|tojson)")|.[]')
 
+		if [[ ${AUTO_FIX:-} == "true" ]]; then
+			if [[ -v linter[autofix] && -n "${linter[autofix]}" ]]; then
+				linter[args]="${linter[autofix]}"
+			else
+				echo "${linter[name]} has no autofix option and has been skipped"
```

Comment:
I don't think this is true; it's still running the linter, just with the default args (i.e. not fixing).  I think we need to unset some things as well for this to be true.  I tested it by building `seiso/goat:ca7b334` and injecting a Hadolint failure;  it failed the Hadolint linter, with no skip message shown.

Reply:
I refactored the global variable AA to a local one. Adding an error test file now skips the linter when an autofix arg is not present.